e2etests/e2etests.js

```diff
@@ -41,12 +41,16 @@ device_provision(hubConnectionString, function (err, provisionedDevices) {
     console.log('Unable to create the devices needed.');
   } else {
     provisionedDevices.forEach(function(deviceToTest) {
-      acknowledgementProtocols.forEach(function (protocolToTest) {
-        device_acknowledge_tests(hubConnectionString, protocolToTest, deviceToTest);
+      if (deviceToTest.authenticationDescription !== 'shared private key') {
```

Comment:
I do not dispute that this area is ripe for rethinking.  That however is NOT what this pull request is about.  I would of course be happy to discuss this with you.  There are certainly good arguments to be made for simplification to especially foster faster e2e test runs.

---
In reply to: [128657867](https://github.com/Azure/azure-iot-sdk-node/pull/61#discussion_r128657867) [](ancestors = 128657867)